addons/im_livechat/static/src/xml/im_livechat_backend.xml

```diff
@@ -7,7 +7,7 @@
             <t t-set="disableAddThread" t-value="true"/>
             <t t-call="mail.discuss.SidebarTitle">
                 <t t-set="title">Livechat</t>
-                <t t-set="icon" t-value="fa-comments"/>
+                <t t-set="icon" t-value="'fas fa-comments'"/>
```

Comment:
This icon is "bugged" since so long that not having the icon is somewhat the expected behavior now I think. To double check with PO that adding back the icon is not creating too much noise in the discuss interface (especially for livechat where all chats have also the same icon because there is no status icon for public users).

Reply:
You can remove them.